### DISCUSSION TITLE
Scoliosis and mattress

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#338] [General]
Hey guys!
Curious how many of you have scoliosis and what kind of mattress do you have?
I am looking forward to buy a new one for my bed and absolutely lost in all the thousands of different mattresses available on the market.
Would highly appreciate some tips on this matter.

**Comment by hernessteve** [comment]
Any good quality mattress works just fine with the scoliosis

**Comment by ellielo** [comment]
Got myself a purple mattress and honestly that is the best thing I ever had

**Comment by hernessteve** [comment]
Congrats! Good quality mattress is one of the best thing you can do for your health

**Comment by MirkosMangle** [comment]
Avoiding stress is indeed crucial for maintaining overall well-being. Thanks to the assistance I received from Academized [https://academized.com/write-my-research-paper](https://academized.com/write-my-research-paper) I've been able to effectively manage my academic workload without feeling overwhelmed. Their support has allowed me to approach my studies with a clearer mind and a more relaxed attitude. As a result, I feel rested and focused, free from the anxiety that can often accompany academic deadlines and responsibilities. Prioritizing my mental health and seeking help when needed has been instrumental in achieving a better balance between academic success and personal well-being.

**Comment by OStebbins215** [comment]
Hey guys!
Curious how many of you have scoliosis and what kind of mattress you have. I’m looking to buy a new one and am lost in the thousands of options. Feels like choosing the right one is almost as tricky as picking the best [custom MVP software development](https://www.urlaunched.com/) service—so many factors to consider! Would highly appreciate some tips on this matter.
